js/widgets/serializers/widget.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2023 ipyforcegraph contributors.
+ * Distributed under the terms of the Modified BSD License.
+ */
+import { WidgetModel, unpack_models } from '@jupyter-widgets/base';
+
+export const widget_serialization = {
+  deserialize: unpack_models,
+  serialize: (models: WidgetModel[]): string[] => {
```

Comment:
this addresses the issue (also seen elsewhere) around the most recent `@jupyter-widgets/base` being more strict about its serialization guarantees. Might get added upstream, but :shrug:.